src/main/java/io/spokestack/spokestack/tts/TTSManager.java

```diff
@@ -254,7 +255,9 @@ public void eventReceived(@NonNull TTSEvent event) {
         switch (event.type) {
             case AUDIO_AVAILABLE:
             case ERROR:
-                this.synthesizing = false;
+                synchronized (lock) {
```

Comment:
the unconditional write here doesn't need to be synchronized, since we know that we must be in the synthesizing state, and no other threads can logically enter this block concurrently

Reply:
I wondered about that, but I wanted to avoid concurrent calls to the event handler and `synthesize` from preventing the processing of the next request. That's impossible, though, since `processQueue` would then be called by both threads.